component/forms/Details.md

```diff
@@ -1,12 +1,12 @@
-## Version 5.1.0
+## Version 5.1.1
 
 HockeySDK-Xamarin implements support for HockeyApp in your iOS and Android applications.
 
 The following features are currently supported:
 
 1. **Collect crash reports:** If your app crashes, a crash log is written to the device's storage. If the user starts the app again, they will be asked to submit the crash report to HockeyApp. This works for both beta and live apps, i.e. those submitted to the App Store. Crash logs contain viable information for you to help resolve the issue. Furthermore, you as a developer can add additional information to the report as well.
 
-2. **User Metrics:** Understand user behavior to improve your app. Track usage through daily and monthly active users. Monitor crash impacted users. Measure customer engagement through session count. You can also track custom events and view the aggregate results on the HockeyApp dashboard. On Android, this feature requires a minimum API level of 14 (Android 4.x Ice Cream Sandwich).
```

Comment:
Just checking, the statement regarding Android can be removed because the SDK requires min API level 14, right?